server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndexer.java

```diff
@@ -90,7 +91,7 @@ public void deleteProject(String uuid) {
     esClient
       .prepareDelete(INDEX_TYPE_PROJECT_MEASURES, uuid)
       .setRouting(uuid)
-      .setRefresh(true)
+      .setRefresh(REFRESH_IMMEDIATE) // ES 5: change to setRefreshPolicy
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)